packages/shared/src/hooks/useActiveNav.tsx

```diff
@@ -44,8 +38,7 @@ export default function useActiveNav(activeFeed: AllFeedPages): UseActiveNav {
 
     if (!isLaptop) {
       homePages.push(OtherFeedPage.Bookmarks);
-
-      if (!notificationsNavBar && !isMobile) {
+      if (!isMobile) {
```

Comment:
Is this the correct eval? this would always be false with the other condition now that notificationsNavBar is true since it won.

Reply:
It needs to be for tablet only basically 😅
Just easier this way to not break too many things.